DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ResourceRestBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * Returns the config tree builder.
+     *
+     * @return TreeBuilder
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $treeBuilder->root('cmf_resource_rest')
+            ->children()
+                ->arrayNode('payload_alias_map')
```

Comment:
Yes, that XML is invalid. It is using an undefined `article` element. It has to be:

````xml
<?xml version="1.0" charset="UTF-8" ?>
<container xmlns="http://symfony.com/schema/dic/services"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns:cmf-resource-rest="http://cmf.symfony.com/schema/dic/cmf_resource_rest"
    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
        http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
    <cmf-resource-rest:config>
        <cmf-resource-rest:payload-alias-map alias="article"
            type="Namespace\Article"
            repository="doctrine_phpcr_odm"
        />
    </cmf-resource-rest:config>
</container>
````